native/SampleApps/SmartSyncExplorerSwift/SmartSyncExplorerSwift/AppDelegate.swift

```diff
@@ -0,0 +1,268 @@
+ are permitted provided that the following conditions are met:
+ * Redistributions of source code must retain the above copyright notice, this list of conditions
+ and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright notice, this list of
+ conditions and the following disclaimer in the documentation and/or other materials provided
+ with the distribution.
+ * Neither the name of salesforce.com, inc. nor the names of its contributors may be used to
+ endorse or promote products derived from this software without specific prior written
+ permission of salesforce.com, inc.
+ 
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+ IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+ WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import UIKit
+import SalesforceSwiftSDK
+import MobileCoreServices
+import SalesforceSDKCore.SFPushNotificationManager
+import SalesforceSDKCore.SFDefaultUserManagementViewController
+import SalesforceSDKCore.SalesforceSDKManager
+import SalesforceSDKCore.SFUserAccountManager
+import SalesforceSDKCore.SFSDKAppConfig
+import SalesforceSDKCore.SFSDKWindowManager
+import SmartSync.SmartSyncSDKManager
+import SalesforceAnalytics.SFSDKDatasharingHelper
+import SalesforceAnalytics.NSUserDefaults_SFAdditions
+import SmartSyncExplorerCommon.SmartSyncExplorerConfig
+
+
+// Fill these in when creating a new Connected Application on Force.com
+//let RemoteAccessConsumerKey = "3MVG9Iu66FKeHhINkB1l7xt7kR8czFcCTUhgoA8Ol2Ltf1eYHOU4SqQRSEitYFDUpqRWcoQ2.dBv_a1Dyu5xa"
+//let OAuthRedirectURI        = "testsfdc:///mobilesdk/detect/oauth/done"
+
+let RemoteAccessConsumerKey = "3MVG9g9rbsTkKnAWkkUnzOrokwAGv9LB5W9tdPJEcQM9xMhNTQnnYHvCwjnjc3zQG5hQ_krRCb52wn0TU2kZH"
+let OAuthRedirectURI        = "com.salesforce.mobilesdk.sample.smartsyncexplorer://oauth2/success"
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    override init() {
+        super.init()
+        
+        guard let config = SmartSyncExplorerConfig.sharedInstance() else { return }
+        SFSDKDatasharingHelper.sharedInstance().appGroupName = config.appGroupName
+        SFSDKDatasharingHelper.sharedInstance().appGroupEnabled = config.appGroupsEnabled
+        
+        SalesforceSwiftSDKManager.setInstanceClass(SmartSyncSDKManager.self)
```

Comment:
You should use `SalesforcSwiftSDKManager.self` instead of `SmartSyncSDKManager.self`.